.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     steps:
       - checkout
       - run: npm config set prefix "$HOME/.local"
-      - run: npm i -g origami-build-tools@^7
+      - run: npm i -g origami-build-tools@^8.2.2
```

Comment:
Now we have renovate/dependabot, should we make obt a devdependency on origami components instead of assuming it is installed globally and having to do this in ci?

Reply:
Hmmm, I can't think of a good reason off the top of my head not to do that.

Reply:
@chee & @rowanmanning, what do you think?

Reply:
apparently i was on board with this